src/components/navigation/document-tab-panel.tsx

```diff
@@ -38,22 +38,22 @@ export class DocumentTabPanel extends BaseComponent<IProps, IState> {
     const { documentView, tabSpec, onTabClick, onDocumentClick } = this.props;
     const { tabIndex } = this.state;
     const { user } = this.stores;
-    const leftTabSpecs = this.stores.appConfig.leftTabs.tabSpecs;
-    const leftTabSpec = leftTabSpecs.find(spec => spec.tab === tabSpec.tab);
+    const contentTabSpecs = this.stores.appConfig.contentTabs.tabSpecs;
+    const contentTabSpec = contentTabSpecs.find(spec => spec.tab === tabSpec.tab);
```

Comment:
```suggestion
    const contentTabSpec = appConfig.getContentTabSpec(tabSpec.tab);
```
There's an accessor now.